source/rust_verify/example/trigger_loops.rs

```diff
@@ -0,0 +1,81 @@
+// rust_verify/tests/example.rs ignore
```

Comment:
Can you add a comment to this file saying that users should expect it to timeout?
`ignore` is the right thing here, but if folks arrive here while browsing for examples it'd be nice if they were warned that this explodes.